src/EnergyPlus/HeatRecovery.hh

```diff
@@ -680,63 +470,34 @@ namespace HeatRecovery {
-    Real64 CurrentEndTimeLast = 0.0;  // end time of time step for last simulation time step
-    std::string OutputChar2;          // character string for warning messages
-    std::string OutputCharLo2;        // character string for warning messages
-    std::string OutputCharHi2;        // character string for warning messages
-    std::string CharValue2;           // character string for warning messages
-    Real64 TimeStepSysLast2 = 0.0;    // last system time step (used to check for downshifting)
-    Real64 CurrentEndTime2 = 0.0;     // end time of time step for current simulation time step
-    Real64 CurrentEndTimeLast2 = 0.0; // end time of time step for last simulation time step
-    std::string OutputChar3;          // character string for warning messages
-    std::string OutputCharLo3;        // character string for warning messages
-    std::string OutputCharHi3;        // character string for warning messages
-    std::string CharValue3;           // character string for warning messages
-    Real64 TimeStepSysLast3 = 0.0;    // last system time step (used to check for downshifting)
-    Real64 CurrentEndTime3 = 0.0;     // end time of time step for current simulation time step
-    Real64 CurrentEndTimeLast3 = 0.0; // end time of time step for last simulation time step
-    std::string OutputChar4;          // character string for warning messages
-    std::string OutputCharLo4;        // character string for warning messages
-    std::string OutputCharHi4;        // character string for warning messages
-    std::string CharValue4;           // character string for warning messages
-    Real64 TimeStepSysLast4 = 0.0;    // last system time step (used to check for downshifting)
-    Real64 CurrentEndTime4 = 0.0;     // end time of time step for current simulation time step
-    Real64 CurrentEndTimeLast4 = 0.0; // end time of time step for last simulation time step
-    std::string OutputChar5;          // character string for warning messages
-    std::string OutputCharLo5;        // character string for warning messages
-    std::string OutputCharHi5;        // character string for warning messages
-    Real64 TimeStepSysLast5 = 0.0;    // last system time step (used to check for downshifting)
-    Real64 CurrentEndTime5 = 0.0;     // end time of time step for current simulation time step
-    Real64 CurrentEndTimeLast5 = 0.0; // end time of time step for last simulation time step
-    std::string OutputChar6;          // character string for warning messages
-    std::string OutputCharLo6;        // character string for warning messages
-    std::string OutputCharHi6;        // character string for warning messages
-    Real64 TimeStepSysLast6 = 0.0;    // last system time step (used to check for downshifting)
-    Real64 CurrentEndTime6 = 0.0;     // end time of time step for current simulation time step
-    Real64 CurrentEndTimeLast6 = 0.0; // end time of time step for last simulation time step
+    ErrorTracker2 error1;
```

Comment:
I was not interested in finding more meaningful names for these now.  But:

```
    ErrorTracker2 error2;
```

is better than:

```
    std::string OutputChar2;          // character string for warning messages
    std::string OutputCharLo2;        // character string for warning messages
    std::string OutputCharHi2;        // character string for warning messages
    std::string CharValue2;           // character string for warning messages
    Real64 TimeStepSysLast2 = 0.0;    // last system time step (used to check for downshifting)
    Real64 CurrentEndTime2 = 0.0;     // end time of time step for current simulation time step
    Real64 CurrentEndTimeLast2 = 0.0; // end time of time step for last simulation time step
```